schemas/appeal-s78.schema.json

```diff
@@ -0,0 +1,1078 @@
+    },
+    "ownershipCertificateSubmitted": {
+      "type": ["boolean", "null"],
+      "description": "Indicates that an ownership certificate has been submitted",
+      "$comment": "Only available in S78 cases"
+    },
+    "designAccessStatementProvided": {
+      "type": ["boolean", "null"],
+      "description": "Indicates that a design and access statement has been provided",
+      "$comment": "Only available in S78 cases"
+    },
+    "newPlansDrawingsProvided": {
+      "type": ["boolean", "null"],
+      "description": "Indicates that new plans or drawings have been provided",
+      "$comment": "Only available in S78 cases"
+    },
+    "planningObligation": {
+      "type": ["boolean", "null"],
+      "description": "Indicates the existence of a planning obligation",
+      "$comment": "Only available in S78 cases"
+    },
+    "statusPlanningObligation": {
+      "type": ["string", "null"],
+      "description": "The planning obligation information, if available",
+      "$comment": "Only available in S78 cases"
+    },
+    "appellantFinalCommentDetails": {
+      "type": ["string", "null"],
+      "description": "The final comments details indicated by the appellant",
+      "$comment": "Only available in S78 cases"
+    },
+    "appellantWitnesses": {
+      "type": ["boolean", "null"],
+      "description": "Indicates if there are any witnesses",
+      "$comment": "Only available in S78 cases"
+    },
+    "affectsScheduledMonument": {
+      "type": ["boolean", "null"],
+      "description": "Indicates that a scheduled monument is affected",
+      "$comment": "Only available in S78 cases"
+    },
+    "hasProtectedSpecies": {
+      "type": ["boolean", "null"],
+      "description": "Indicates the existence of protected species",
+      "$comment": "Only available in S78 cases"
+    },
+    "isAonbNationalLandscape": {
+      "type": ["boolean", "null"],
+      "description": "Indicates an area of outstanding beauty (National Landscape)",
+      "$comment": "Only available in S78 cases"
+    },
+    "designatedSitesNames": {
+      "type": ["string", "null"],
+      "description": "The designated site names"
+    },
+    "hasTreePreservationOrder": {
+      "type": ["boolean", "null"],
+      "description": "Indicates a TPO (tree preservation order)",
+      "$comment": "Only available in S78 cases"
+    },
+    "isGypsyOrTravellerSite": {
+      "type": ["boolean", "null"],
+      "description": "Indicates a traveller site",
+      "$comment": "Only available in S78 cases"
+    },
+    "isPublicRightOfWay": {
+      "type": ["boolean", "null"],
+      "description": "Indicates a PROW (public right of way)",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaEnvironmentalImpactSchedule": {
+      "type": ["string", "null"],
+      "description": "The impact schedule from EIA",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaDevelopmentDescription": {
+      "type": ["string", "null"],
```

Comment:
we have:
["agriculture-aquaculture", "change-extensions", "chemical-industry", "energy-industry", "extractive-industry", "food-industry", "infrastructure-projects", "mineral-industry", "other-projects", "production-processing-of-metals", "rubber-industry", "textile-industries", "tourism-leisure"]